beater/beater.go

```diff
@@ -394,6 +395,19 @@ func (s *serverRunner) run() error {
 	s.acker.Open()
 	pipeline := pipetool.WithACKer(s.pipeline, s.acker)
 
+	if eac := os.Getenv("ELASTIC_AGENT_CLOUD"); eac == "" && s.config.JavaAttacherConfig.Enabled {
+		// Not running on ECE/ESS, start the java attacher
+		go func() {
+			attacher, err := javaattacher.New(s.config.JavaAttacherConfig)
+			if err != nil {
+				s.logger.Errorf("java attacher error: %v", err)
```

Comment:
shouldn't be returning here or something?

Reply:
yep, whoops :sweat_smile: